zerver/views/users.py

```diff
@@ -203,6 +211,46 @@ def reactivate_user_backend(
 )
 
 
+def notify_users_on_updated_user_profile(
+    acting_user: UserProfile,
+    recipient_user: UserProfile,
+    *,
+    old_full_name: Optional[str],
+    new_full_name: Optional[str],
+) -> None:
+    realm = acting_user.realm
+    mention_backend = MentionBackend(realm.id)
+
+    notification = []
+    sender = get_system_bot(settings.NOTIFICATION_BOT, realm.id)
+
+    if recipient_user.is_bot or acting_user == recipient_user:
+        return
+
+    with override_language(recipient_user.default_language):
+        message = _("{user_full_name} has made the following changes to your profile.").format(
+            user_full_name=silent_mention_syntax_for_user(acting_user),
+        )
+        if old_full_name and new_full_name:
+            message += _(
+                "\n- **Old `name`:** {old_full_name}\n- **New `name`:** {new_full_name}"
+            ).format(
+                old_full_name=old_full_name,
+                new_full_name=new_full_name,
+            )
+
+    notification.append(
+        internal_prep_private_message(
```

Comment:
I think we should use [`internal_send_private_message`](https://github.com/zulip/zulip/blob/784622ee5d3e5bca4db9ebafba748acd797b427d/zerver/actions/message_send.py#L1614) directly.